pyk/src/kllvm/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import kllvm
+from .kllvm import ast
```

Comment:
Is this the suggested way for wrapping native `pybind11` modules? Whe I try to import the module from an interpreter, I get the following error.

```
In [1]: import kllvm
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 import kllvm

File ~/.local/lib/python3.8/site-packages/kllvm/__init__.py:1, in <module>
----> 1 from . import kllvm
      2 from .kllvm import ast

ImportError: cannot import name 'kllvm' from partially initialized module 'kllvm' (most likely due to a circular import) (/home/ttoth/.local/lib/python3.8/site-packages/kllvm/__init__.py)
```

Reply:
Did you run `make deps` in the `pyk` directory beforehand?

Reply:
(i.e. is there a shared library `kllvm.platform-os-etc.so` beside `src/kllvm/__init__.py`?)

Reply:
Currently, both of the following work:

```python3
from kllvm.kllvm import ast
from kllvm import ast
```

Did you consider putting this in `__init__.py` instead?

```python3
from .kllvm import *
```

Reply:
If that doesn't solve the circular import problem of `ipython`, an other solution to consider is to rename the native module to `_kllvm`. (That also communicates to users that the module is not supposed to be imported directly.)